packages/astro/src/default/layouts/Layout.astro

```diff
@@ -38,7 +38,8 @@ const baseURL = import.meta.env.BASE_URL;
       }
     </script>
     <script>
-      import * as builtInSwap from 'node_modules/astro/dist/transitions/swap-functions';
+      // @ts-ignore -- untyped
```

Comment:
We can add an entry for that module in `env-default.d.ts` to avoid the `//@ts-ignore`.

(we used to have a type error here though that we ignored so maybe this is fine)

Reply:
I think it's better to keep `@ts-ignore`. By defining types in declaration file, we would need to somehow try to keep them up-to-date with Astro's implementation.

Reply:
Oh I was thinking of declaring the module as exporting `any` like we do for CSS modules.

Reply:
But that would require specifying each named export in the declaration, right? There's no way to say dynamically "all named exports from here are any".

Reply:
You don't have to. You can do like what we do for CSS modules:

```ts
declare module 'tutorialkit:astro-swap-functions';
```

Reply:
Nice - happy to learn new tricks like this! 💯